### PR TITLE
Making BIRT compile with Maven 3.5+ again...

### PR DESCRIPTION
**Pull request by zaplatynski**
... and enable Travis builds with Maven toolchains configuration: https://travis-ci.org/zaplatynski/birt

### CODE REVIEW

**Comment by eryabitskiy-acx** [comment]
Hallöchen,

Looks like your pull request is very similar to mine:
https://github.com/eclipse/birt/pull/461
same Maven poms cleanup and etc.
Btw cool idea about Travis-Ci! 👍 
My pull request is "opened 23 days" ago. And still no any response from committers :(

It was so disappointing... That we have created a fork and doing our small parallel development.
https://github.com/ascendix/birt

Maven build is fixed
Tests are fixed
garbage in src is removed
build is cleaned up
Travis-Ci is there.

Feel free to join.))

**Comment by yulinwang** [comment]
This is similar to #461, which I have replied before. It's not good because there are too many hardcoded versions in parent. This increases the effort to upgrade version.
If you can have a separate changelist for travis-CI, I'm glad to accept that, but please do not use your URL: https://travis-ci.org/zaplatynski/birt

**Comment by yulinwang** [comment]
In fact BIRT has own CI in: https://hudson.eclipse.org/birt/. Not sure if you can access it.

**Comment by eryabitskiy-acx** [comment]
@yulinwang Nope, there are no replies in https://github.com/eclipse/birt/pull/461 from you.

**Comment by pwolf1985** [comment]
Guten Tag,

vielen Dank für Ihre Nachricht.

Ich bin am 06.09.18 nicht erreichbar.

Aus Vertraulichkeitsgründen wurde Ihre E-Mail nicht automatisch weitergeleitet. Vielen Dank für Ihr Verständnis.

Für SalesFront Themen wenden Sie sich bitte an Support@salesfrontapp.com!

Rund um das Thema TelePhone kontaktieren Sie bitte TelePhoneSupport@wkg-software.de!

In sonstigen dringenden Fällen wenden Sie sich bitte an Herrn Markus Wiegand (wiegand@wkg-software.de) 

Mit freundlichen Grüßen

Patrick Wolf
M.Sc. Informatik
Entwicklungsleiter
 
W.K.G Software GmbH
Hindenburgstrasse 38, 30175 Hannover, Germany
Geschäftsführer: Dipl.-Kfm. Jochen Wunder
HRB 60250 Amtsgericht Hannover, USt.-ID: DE 813 392 760
 
Phone: +49 511 67691-0     Email: info@wkg-software.de
Fax:     +49 511 67691-111 Web:  www.wkg-software.de

**Comment by zaplatynski** [comment]
@yulinwang Sure I can change that URL if you like but you have to enable travis-ci for your repo.

**Comment by zaplatynski** [comment]
@eryabitskiy-acx I saw your PR after I submitted mine :( 
Glad that I you give some idea with Travis-CI. How did you fix the toolchains.xml? You don't need it?

**Comment by zaplatynski** [comment]
@yulinwang Then add this link to the readme so that people can see it if they want to.

To the reason that the update of the parent version has to be done manually... that is not true because there a maven version plugin which does that for you... but if you have a mixture of parent versions within the project (and this is working against the Maven way or the way Maven intends) then nobody is able to compile it because Maven central is missing the the 0.0.1-SNAPSHOT.

**Comment by yulinwang** [comment]
@eryabitskiy-acx Sorry I forgot to submit the whole review but only submitted the inline review. Now it should be visible.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# BIRT
+# BIRT [![Build Status](https://travis-ci.org/zaplatynski/birt.svg?branch=master)](https://travis-ci.org/zaplatynski/birt)
```
Comment (yulinwang) [inline README.md]: Could you change to: https://travis-ci.org/eclipse/birt?

**Comment by eryabitskiy-acx** [comment]
@zaplatynski I have just removed toolchain plugin. 
**Since Eclipse compatible with only Java 8 starting from Eclipse 4.6 (Neon).**
I don't see any reason to support older versions of Java.
https://wiki.eclipse.org/Eclipse/Installation

**Pull request merged**